src/Extend/SitesLinkedToMagentoStores.php

```diff
@@ -20,4 +22,38 @@ public function findByMageRunCode($code)
     {
         return collect($this->sites)->get($code);
     }
+
+    protected function getSavedSites()
+    {
+        return Cache::driver('array')->rememberForever('statamic_sites', function () {
+            $sites = [];
+            $stores = Rapidez::getStores();
+            $configModel = config('rapidez.models.config');
+
+            foreach ($stores as $store) {
+                if (config('rapidez.statamic.sites.' . $store['code'] . '.attributes.disabled')) {
+                    continue;
+                }
+
+                Rapidez::setStore($store['store_id']);
+
+                $locale = $configModel::getCachedByPath('general/locale/code');
+                $lang = explode('_', $locale)[0] ?? '';
+                $url = $configModel::getCachedByPath('web/secure/base_url');
+
+                $sites[$store['code']] = [
+                    'name' => $store['name'] ?? $store['code'],
+                    'locale' => $locale,
+                    'lang' => $lang,
+                    'url' => $url,
+                    'attributes' => [
+                        'magento_store_id' => $store['store_id'],
+                        'group' => $store['website_code'] ?? ''
+                    ]
+                ];
+            }
+
+            return $sites ?: $this->getFallbackConfig();
```

Comment:
The method was changed recently: https://github.com/statamic/cms/pull/10470, do we require the versions where that's changed? https://github.com/statamic/cms/releases/tag/v5.16.0